build.xml

```diff
@@ -121,6 +121,7 @@ xmlns:cs="antlib:com.puppycrawl.tools.checkstyle.ant">
     <property name="ivy.test.lib" value="${build.dir}/test/lib"/>
     <property name="ivy.jdiff.lib" value="${build.dir}/jdiff/lib"/>
     <property name="ivy.javacc.lib" value="${build.dir}/javacc/lib"/>
+    <property name="ivy.owasp.lib" value="${build.dir}/owasp/lib"/>
```

Comment:
iirc the reports are output to build/test/owasp/.... perhaps you should move them here as well rather than spreading around?

Reply:
the reports are output to build/test/owasp/ in order to match what we do with clover
so i felt it was best to follow convention rather than drop the actual report in with the dependencies